web-ui/src/main/resources/catalog/templates/admin/harvest/harvest-settings.html

```diff
@@ -238,9 +238,9 @@ <h4 data-ng-show="h.deleted == 'false'" class="timeline-title">
                     {{k}}: {{v}}
                   </li>
                 </ul>
-
+                <!-- in some scenario logfile returns an array of files, take first -->
                 <a data-ng-show="h.info[0].logfile"
-                   href="admin.harvester.log?file={{h.info[0].logfile}}" target="_blank"
+                   href="admin.harvester.log?file={{ encodeURI((h.info[0].logfile | asArray)[0]) }}" target="_blank"
```

Comment:
This change may no longer be needed, server now checks to ensure only one logfile is returned (and ensures the file exists and is readable also).